src/pages/login/login.ts

```diff
@@ -33,7 +37,12 @@ export class LoginPage {
         })
     }
     signin(formData) {
-        this.consignmentCheck();
+        this._sqlProvider.openDb().then((db: SQLiteObject) => {
```

Comment:
there should be a seperate service for this. don't place any sql code in components

Reply:
ok will remove this when i will create service for this